src/libs/API.js

```diff
@@ -528,6 +528,19 @@ function Report_UpdateLastRead(parameters) {
     return Network.post(commandName, parameters);
 }
 
+/**
+ * @param {Object} parameters
+ * @param {Number} parameters.reportID
+ * @param {Number} parameters.reportActionID
+ * @param {String} parameters.reportComment
+ * @returns {Promise}
+ */
+function Report_EditComment(parameters) {
+    const commandName = 'Report_EditComment';
+    requireParameters(['accountID', 'reportID', 'reportActionID', 'reportComment'], parameters, commandName);
```

Comment:
I don't believe `accountID` is required for the [Report_EditComment API command](https://github.com/Expensify/Web-Expensify/blob/30dcc2044ce8f781cd9aff6cd823a1b2f168c631/api.php#L586)